src/js/components/toolbar/AddMarks.jsx

```diff
@@ -54,7 +55,7 @@ var AddMarksTool = React.createClass({
         {marksArray.map(function(markType, i) {
           return (
             <li key={markType}
-              onClick={this.props.addMark.bind(null, markType, closestContainerId)}>
+              onClick={this.props.addMark.bind(this, markType, closestContainerId)}>
```

Comment:
Hmm, but I'm curious why `closestContainerId` isn't set the right thing? From the looks of it, we don't clear out `inspectors.selected` in the redux store on `MARK_DELETE`. As a result, `closestContainerId` isn't set to the `sceneId`. Would that be a better fix?